src/libraries/System.Net.Security/src/System/Net/Security/SslStream.IO.cs

```diff
@@ -56,8 +56,8 @@ private void CloseInternal()
             // block potential future read and auth operations since SslStream is disposing.
             // This leaves the _nestedRead = 1 and _nestedAuth = 1, but that's ok, since
             // subsequent operations check the _exception sentinel first
-            if (Interlocked.Exchange(ref _nestedRead, 1) == 0 &&
-                Interlocked.Exchange(ref _nestedAuth, 1) == 0)
+            if (Interlocked.Exchange(ref _nestedRead, 2) == 0 &&
```

Comment:
Nit: now that it's no longer just binary 0 and 1, it'd be nice to define some named consts for the values

Reply:
I added Enum for it.